fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/impl/resource/loader/FabricModResourcePack.java

```diff
@@ -61,6 +74,48 @@ public InputSupplier<InputStream> openRoot(String... pathSegments) {
 		return null;
 	}
 
+	private String generateMetadataJson() {
+		record PackMetadata(PackResourceMetadata pack, Optional<ResourceFilter> filter) {
+			static final Codec<PackMetadata> CODEC = RecordCodecBuilder.create(instance ->
+					instance.group(
+							PackResourceMetadata.CODEC.fieldOf("pack").forGetter(PackMetadata::pack),
+							ResourceFilterAccessor.getCodec().optionalFieldOf("filter").forGetter(PackMetadata::filter)
+					).apply(instance, PackMetadata::new));
+		}
+
+		final var resourceMetadata = new PackResourceMetadata(
+						Text.translatableWithFallback("pack.description.modResources", "Mod resources."),
+						SharedConstants.getGameVersion().getResourceVersion(type),
+				Optional.empty()
+		);
```

Comment:
```suggestion
		final var resourceMetadata = new PackResourceMetadata(
				Text.translatableWithFallback("pack.description.modResources", "Mod resources."),
				SharedConstants.getGameVersion().getResourceVersion(type),
				Optional.empty()
		);
```
4 tabs seems a bit excessive

Reply:
I agree, but checkstyle doesnt.

Reply:
It's also inconsistent with the final line of parameters, which only uses 2 tabs 🤔

Reply:
Oh, I see what you mean. In idea it was impossible to see this... 
<img width="1020" alt="Screenshot 2023-09-11 at 22 33 26" src="https://github.com/FabricMC/fabric/assets/4324090/f4c7bd3e-2231-4417-9355-e444ebfb54a7">